tests/unit/test_model_helper.py

```diff
@@ -0,0 +1,39 @@
+from typing import List, Optional, Tuple
+from unittest import TestCase
+
+from openslides_backend.models.helper import calculate_inherited_groups_helper
+
+
+class ModelHelper(TestCase):
```

Comment:
Also rename this class and file to accurately reflect what is being tested - the `calculate_inherited_groups_helper`

Reply:
Renamed.